src/test/scala/com/itechart/ny_accidents/service/WeatherMappingServiceTest.scala

```diff
@@ -0,0 +1,5 @@
+package com.itechart.ny_accidents.service
+
+class WeatherMappingServiceTest {
```

Comment:
?

Reply:
For the future. it can be deleted now. for fair, I just forget about it

Reply:
Please, delete or add todo

Reply:
Done